.github/scripts/check_labels.py

```diff
@@ -75,16 +76,20 @@ def main() -> None:
     org, project = repo.gh_owner_and_name()
     pr = GitHubPR(org, project, args.pr_num)
 
+    needs_labels = False
     try:
-        if not has_required_labels(pr):
+        needs_labels = not has_required_labels(pr)
+        if needs_labels:
+            exit_code = 1
             print(ERR_MSG)
             add_comment(pr)
-            exit(1)
         else:
             delete_comments(pr)
     except Exception as e:
         pass
 
+    exit(needs_labels)
```

Comment:
Did you mean `exit(exit_code)` here and `exit_code = 0` on line 79?

Reply:
🤦‍♂️ forgot to remove some dead code.  `exit()` converts boolean values into 0/1